src/current/v23.1/data-domiciling.md

```diff
@@ -80,90 +135,238 @@ ALTER DATABASE movr ADD REGION "us-west1";
+(1 row)
 ~~~
 
-This output shows that the `movr` database has ranges out of compliance, which you saw previously. Unfortunately, this output does not contain the table or index names due to a current limitation of the replication reports: non-voting replicas are not associated with any tables or indexes by the reports.
-
 ### Step 4. Apply stricter replica placement settings
 
-To ensure that data on EU-based users, vehicles, etc. from [`REGIONAL BY ROW` tables]({% link {{ page.version.version }}/regional-tables.md %}#regional-by-row-tables) is stored only on EU-based nodes in the cluster, you must disable the use of [non-voting replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#non-voting-replicas) on all of the [regional tables]({% link {{ page.version.version }}/regional-tables.md %}) in this database. You can do this using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`]({% link {{ page.version.version }}/alter-database.md %}#placement) statement.
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="super-region">Use <code>ADD SUPER REGION</code></button>
+  <button class="filter-button page-level" data-scope="placement-restricted">Use <code>PLACEMENT RESTRICTED</code></button>
+</div>
 
-To use this statement, you must set the `enable_multiregion_placement_policy` [session setting]({% link {{ page.version.version }}/set-vars.md %}) or the `sql.defaults.multiregion_placement_policy.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+<div class="filter-content" markdown="1" data-scope="super-region">
+
+To ensure that data on EU-based users, vehicles, etc. from [`REGIONAL BY ROW` tables]({% link {{ page.version.version }}/regional-tables.md %}#regional-by-row-tables) is stored only on EU-based nodes in the cluster, you can use [Super regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions). This will ensure that [regional tables]({% link {{ page.version.version }}/table-localities.md %}#regional-tables) and [regional by row tables]({% link {{ page.version.version }}/table-localities.md %}#regional-by-row-tables) whose [home regions]({% link {{ page.version.version }}/table-localities.md %}) are in the super region will have all of their [replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) stored _only_ in regions that are members of the super region.
+
+To use this statement, you must set the `enable_super_regions` [session setting]({% link {{ page.version.version }}/set-vars.md %}) or the [`ALTER ROLE ALL SET sql.defaults.multiregion_placement_policy.enabled = on` statement]({% link {{ page.version.version }}/alter-role.md %}#set-default-session-variable-values-for-all-users):
```

Comment:
nit: this should either say `ALTER ROLE ALL SET enable_multiregion_placement_policy = true` OR say `SET CLUSTER SETTING sql.defaults.multiregion_placement_policy.enabled = true`.

(sql.defaults.multiregion_placement_policy.enabled is the cluster setting which controls the default value for the enable_multiregion_placement_policy session setting)

but also, is `enable_multiregion_placement_policy` even the right setting here? that controls the PLACEMENT syntax. perhaps you meant to use `SET CLUSTER SETING sql.defaults.super_regions.enabled = true`?

Reply:
Ah you're right the only setting in this example should be `enable_super_regions`, I think this was a copypasta error on my part

I have updated this sentence to recommend doing `ALTER ROLE ALL SET enable_super_regions = on`, and removed mention of that other cluster setting which was incorrect for super regions

(aside: I have also updated this PR to add some simple docs on the session variables page for `enable_super_regions` since we did not have any there (probably b/c this was experimental or something when first added moons ago)

Reply:
PS thanks for the review @rafiss !!!!